### PR TITLE
Portfolio Popup

### PR DESCRIPTION
**Pull request by TekyRey**
### **Portfolio Popup**
In this project we:

- made the portfolio cards dynamic using JavaScript arrays.
- created and designed the popup modal.
- used JavaScript to load the popup modal

### CODE REVIEW

**Review by tresorsawasawa — COMMENTED** [review]
# STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @TekyRey 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project which needs pair-programming but it was done with one collaborator. Please, make sure your coding partner has some contributions before you requested again for a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

---
Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

---
_Invalid Code Review Request **Does not count** into the code reviews limit._

File: javascript/index.js [hunk 1]
```diff
@@ -15,3 +15,128 @@ mobileMenuItems.forEach((item) => {
...
+
+const portfolio = document.querySelector('#portfolio');
+
+const projectArray = [
+  {
+    mobileImg: 'images/tonic-section.svg',
+    desktopImg: 'images/desktop-snap3.png',
+    cardTitle: 'Tonic',
+    cardDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required',
+    class: 'card',
```
Comment (tufoinnkuo10) [inline javascript/index.js]: - [ ] Great job so far. your list of technologies and your `see live and see source` should be coming from your array- objects as well. this helps to make each of them unique to a particular project. kindly fix that.

File: javascript/index.js [hunk 2]
```diff
@@ -15,3 +15,128 @@ mobileMenuItems.forEach((item) => {
...
+  {
+    mobileImg: 'images/tonic-section.svg',
+    desktopImg: 'images/desktop-snap3.png',
+    cardTitle: 'Tonic',
+    cardDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required',
+    class: 'card-2',
+  },
+];
+
+portfolio.innerHTML = projectArray.map((project) => {
```
Comment (tufoinnkuo10) [inline javascript/index.js:89]: - [ ] please fix the `left,right,up,down` scrolling of your mobile popup, make it full height. also, ensure it looks almost the same as the figma version. 
your version 👇🏾 
Screenshot (159)_LI
figma version 👇🏾 

Screenshot (158)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @TekyRey 

Great job so far for doing the good work. nevertheless, there are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅popup works as expected
✅ cards gotten from array

# Required Changes ♻️
Check the comments under the review

tufoin

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by ishimwezachee — APPROVED** [review]
Hi @TekyRey , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**